src/ubuntu/18.04/android/Dockerfile

```diff
@@ -20,3 +21,24 @@ RUN wget https://dl.google.com/android/repository/android-ndk-r21-linux-x86_64.z
     && unzip android-ndk-r21-linux-x86_64.zip -d /usr/local \
     && mv /usr/local/android-ndk-r21 /usr/local/ndk \
     && rm -f android-ndk-r21-linux-x86_64.zip
+
+# Install Android SDK
+RUN apt-get update \
+    && apt-get install -y \
+        openjdk-8-jdk \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://dl.google.com/android/repository/commandlinetools-linux-6200805_latest.zip \
+    && mkdir -p /usr/local/android-sdk/cmdline-tools \
+    && unzip commandlinetools-linux-6200805_latest.zip -d /usr/local/android-sdk/cmdline-tools \
+    && rm -f commandlinetools-linux-6200805_latest.zip
+
+# Grab the necessary Android SDK packages / tools
+ENV SDK_HOME /usr/local/android-sdk
```

Comment:
this should be called `ANDROID_SDK_ROOT`, it's a common variable recognized by the Android tooling: https://developer.android.com/studio/command-line/variables#envar